website/blog/2024-08-26-how-we-run-python-using-webassembly/index.mdx

```diff
@@ -0,0 +1,91 @@
+# Running Python code with WebAssembly
+
+Language runtimes are nice, but integrating one is an entire story. Let's discover how we managed to integrate the Python runtime within the Metatype ecosystem.
+
+## Why?
+
+You have probably heard of "Function as a Service" or FaaS. In simple terms, FaaS enables anyone to run code in response to events without the hassle of managing the underlying infrastructure needed for microservices.
+
+This capability was initially added into Metatype through Deno workers, which provided ways to execute functions authored in Typescript with ease.
+
+On the other hand, people might also want to write functions in Python, which, as you may know, is very popular, and its standard library can be quite resourceful for this type of use case.
+
+However, integrating a Python runtime isn't as straightforward as integrating Deno.
+
+## What are the constraints?
+
+There are few constraints that had to be met:
+
+1. **Security**: functions should have limited access to the execution evironment
+2. **Speed**: functions should run with low latency
+3. **User-friendliness**: writing a function in one language should be as near identical as writing one in another
+
+## Choosing an approach...
+
+Let's start simple, we want to run Python code, so what do we need?
+
+- A Python runtime
+
+We do not want to be platform-dependent, but how can we achieve that?
```

Comment:
A few more words towards the limitation of the python runtime might be good. We can do this on Deno but not python, why?

Reply:
Good point. We can provide an overview of how we did it with Deno as a parallel.